test/api/CMakeLists.txt

```diff
@@ -1,10 +1,12 @@
-
 function(add_api_test TEST_NAME TEST_SRC TEST_DIR)
     set(NAME test_api_${TEST_NAME})
     add_executable(${NAME} EXCLUDE_FROM_ALL ${TEST_SRC})
     rocm_clang_tidy_check(${NAME})
     target_link_libraries(${NAME} migraphx_c migraphx)
     target_include_directories(${NAME} PUBLIC ../include)
+    if(MIGRAPHX_ENABLE_GPU)
+    target_link_libraries(${NAME} migraphx_gpu)
```

Comment:
This should be done outside of this function.

Reply:
Done.